qemu/tests/qemu_guest_agent.py

```diff
@@ -390,6 +393,265 @@ def gagent_check_set_vcpus(self, test, params, env):
             raise error.TestFail("the vcpu status is not changed as expected")
 
     @error.context_aware
+    def gagent_check_get_time(self, test, params, env):
+        """
+        Execute "guest-get-time" command to guest agent
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        timeout = float(params.get("login_timeout", 240))
+        session = self.vm.wait_for_login(timeout=timeout)
+        os_type = params["os_type"]
+        error.context("get the time of the guest", logging.info)
+        nanoseconds_time = self.gagent.get_time()
```

Comment:
I find declaretion in avocado-framework/avocado-vt#558.
It's too old PR.
I almost forgot about it.